NEWS.Rmd

```diff
@@ -1,7 +1,66 @@
 ---
 output: github_document
 ---
-# madshapR (development version)
+
+# madshapR 1.0.3
+
+## Bug fixes and improvements
+
+Some of the tests were made with another package (Rmonize) which as "madshapR" as
+a dependence.
+
+### Enhance reports
+
+* in visual reports, void confusing changes in color scheme in visual reports.
```

Comment:
"avoid confusing"

Also, not important, but in future, start new lines/sentences consistently with a capital letter.